geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy

```diff
@@ -0,0 +1,42 @@
+package org.orbisgis
+
+import groovy.transform.BaseScript
+import org.orbisgis.datamanager.JdbcDataSource
+import org.orbisgis.datamanagerapi.dataset.ITable
+import org.orbisgis.processmanagerapi.IProcess
+
+@BaseScript Geoclimate geoclimate
+
+/**
+ * This process is used to compute basic statistical operations on a specific variable from a lower scale (for
+ * example the sum of each building volume constituting a block to calculate the block volume)
+ * @return A database table name.
+ * @author Erwan Bocher
+ */
+static IProcess unweightedOperationFromLowerScale() {
+return processFactory.create(
+        "Unweighted statistical operations from lower scale",
+        [inputLowerScaleTableName: String,inputCorrelationTableName: String,inputIdLow: String, inputIdUp: String,
+         inputToTransfo: String,operations: String[], outputTableName: String, datasource: JdbcDataSource],
+        [outputTableName : String],
+        { inputLowerScaleTableName, inputCorrelationTableName, inputIdLow, inputIdUp,
+          inputToTransfo, operations, outputTableName, datasource ->
+            def ops = ["SUM","AVG"]
```

Comment:
count, min, max ???

Reply:
So far (for the indicators we have defined) there is no need for such functions. However, we could add them if you think it is necessary for the future.